zng/docs/spec.md

```diff
@@ -343,7 +372,7 @@ previously defined type, the appropriate typedef control code must
 be re-emitted
 (and note that the typedef may now be assigned a different ID).
 
-### 2.2 ZNG Value Messages
+### 3.2 Value Messages
```

Comment:
Maybe I went a little crazy with the scissors here, but while I was cleaning up the TZNG section, I realized we were being inconsistent with the headers, i.e. redundant references to "ZNG" in headers of sub-sections that had already been described in a higher-level header as being about ZNG. Rather than have lots of redundant ZNG refs in the ZNG headers & redundant TZNG refs in the TZNG section, I opted for consistent brevity.